src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends FigureBase {
+    private int firstSide;
+    private int secondSide;
+
+    public Rectangle(String color, int firstSide, int secondSide) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create separate variables for `firstSide` and `secondSide` and pass them to the constructor for better code readability.